packages/cli/src/Logger.ts

```diff
@@ -0,0 +1,77 @@
+import * as winston from 'winston';
+import config = require('../config');
+
+import {
+	ILogger,
+	LogTypes,
+} from 'n8n-workflow';
+
+export class Logger implements ILogger {
+	private logger: winston.Logger;
+	
+	constructor() {
+
+		const logFormat = winston.format.printf(({ message }) => {
+			return message;
+		}) as winston.Logform.Format;
+		
+		this.logger = winston.createLogger({
+			level: config.get('logs.level'),
+			format: logFormat,
+			transports: [
+				new winston.transports.Console(),
+			],
+		});
+		
+		if (config.get('logs.output') === 'file') {
+			const fileLogFormat = winston.format.combine(
+				winston.format.timestamp(),
+				winston.format.json()
+			);
+			this.logger.add(
+				new winston.transports.File({
+					filename: config.get('logs.file.location'),
+					format: fileLogFormat,
```

Comment:
should not we add some form of log rotation? or is that handled elsewhere?
file transport format supports maxsize and maxFiles to support this https://github.com/winstonjs/winston/blob/master/docs/transports.md#file-transport

Reply:
Hi @mutdmour 

I am being very conservative to adding dependencies and it seems like adding log rotations would require another one.

The thoughts behind this first implementation is to create a basic logging framework that can be improved later, so for this reason I did not add any dependency for log output as all required new dependencies. Transports like elastic, loggly and cloudwatch logs all require more dependencies.

Maybe we can allow users to plug in other transports as they need. What do you think?

Reply:
adding maxsize and maxFiles should not require any new dependencies. File transport used here already supports these https://github.com/winstonjs/winston/blob/master/docs/transports.md#file-transport if I am not mistaken 
I have seen winston log files fill up disk before in a webapp without `maxsize` being set.

Reply:
Thanks for clarifying that @mutdmour .

I just realized that daily rotation would require a new lib but adding these settings do not.

Will apply those changes!